Mathlib/Algebra/BigOperators/Group/Finset.lean

```diff
@@ -2008,6 +2008,10 @@ lemma prod_ite_eq_ite_exists (p : ι → Prop) [DecidablePred p] (h : ∀ i j, p
 
 variable [DecidableEq ι]
 
+@[to_additive]
+lemma prod_ite_mem (s : Finset ι) (f : ι → α) : ∏ i, (if i ∈ s then f i else 1) = ∏ i ∈ s, f i := by
```

Comment:
```suggestion
lemma prod_ite_mem (s : Finset ι) (f : ι → α) : ∏ i, if i ∈ s then f i else 1 = ∏ i ∈ s, f i := by
```
or not?

Reply:
No, doesn't work. `if then else` has very low precedence